Code/GraphMol/RWMol.cpp

```diff
@@ -70,7 +167,8 @@ void RWMol::insertMol(const ROMol &other) {
     for (auto &v : bond_p->getStereoAtoms()) {
       v = newAtomIds[v];
     }
-    addBond(bond_p, true);
+    auto bond_count = addBond(bond_p, true);
+    newBondIds[other.d_graph[*firstB]->getIdx()] = bond_count - 1;
```

Comment:
Can perhaps be simplified as ( and will still work if we ever change the return off addBond)

```
const bool takeOwnership = true;
addBond(bond_p, takeOwnership);
newBondIds[other.d_graph[*firstB]->getIdx()] = bond_p->getIdx();
```

Reply:
Yeah, makes sense